Makefile

```diff
@@ -257,15 +262,15 @@ unit-test:
 	sleep 3
 	docker run -d  --rm --name postgres -e POSTGRES_PASSWORD=admin -e POSTGRES_USER=admin -p 127.0.0.1:5432:5432 postgres:12.3-alpine -c 'max_connections=10000'
 	until PGPASSWORD=admin pg_isready -U admin --dbname postgres --host 127.0.0.1 --port 5432; do sleep 1; done
-	SKIP_UT_DB=1 go test -v $(or ${TEST}, ${TEST}, $(shell go list ./... | grep -v subsystem)) -cover || (docker kill postgres && /bin/false)
+	SKIP_UT_DB=1 go test -v $(or ${TEST}, ${TEST}, $(shell go list ./... | grep -v subsystem)) $(GINKGO_FOCUS_FLAG) -cover || (docker kill postgres && /bin/false)
 	docker kill postgres
 
 test-onprem:
 	INVENTORY=127.0.0.1:8090 \
 	INVENTORY=127.0.0.1:8090 \
 	DB_HOST=127.0.0.1 \
 	DB_PORT=5432 \
-	go test -v ./subsystem/... -count=1 -ginkgo.focus=${FOCUS} -ginkgo.v -timeout 30m
+	go test -v ./subsystem/... -count=1 -$(GINKGO_FOCUS_FLAG) -ginkgo.v -timeout 30m
```

Comment:
Remove the missing `-`

Reply:
no need for that. The flag GINKGO_FOCUS_FLAG is translated to "-ginkgo.focus=${FOCUS}" if the FOCUS env is set or is "" otherwise. In this way we can avoid test failing because a flag is not set.

Reply:
There is a redundant `-` in `test-onprem` before the `$(GINKGO_FOCUS_FLAG)` flag :)